doc/README.OutputSpecs.md

```diff
@@ -0,0 +1,71 @@
+
+Notable exceptions from the MAPL-generated output include:
+* log files [ASCII], 
+* observation-space "ObsFcstAna" data assimilation diagnostics [BINARY], and
+* SMAP L4_SM-specific "aup" data assimilation diagnostics files (available _**only**_ for simulations in EASEv2_M09 tile space) [BINARY].
+
+Output of the latter two sets of files can be turned on/off in the `[NML_INPUT_PATH]/LDASsa_SPECIAL_inputs_ensupd.nml` configuration file, and Matlab readers are available in the 
+[src/Applications/LDAS_App/util](https://github.com/GEOS-ESM/GEOSldas/tree/main/src/Applications/LDAS_App/util) directory.
+
+
+### MAPL HISTORY output
+
+As part of `ldas_setup`, a sample `HISTORY.rc` configuration file is created in the experiment's `./run` directory.  Users specify the desired output by editing `HISTORY.rc`.  
+
+`HISTORY.rc` defines a number of output file "Collections", each of which contains one or more output variables.  Output can be gridded ("2d") or in the native tile space ("1d").  
+
+All variables contained in a given Collection are written:
+* on the same ("2d") grid (if gridded), 
+* at the same frequency, and 
+* with either time-average ("tavg") or instantaneous ("inst") sampling mode.  
+
+In the following example, the `tavg3_2d_lnd_Nx` Collection contains time-average ("tavg"), 3-hourly ("3"), gridded ("2d") data, and the inst1_1d_lfs_Nt Collection contains snapshot/instantaneous ("inst"), 1-hourly, tile-space ("1d") data. 
+```
+COLLECTIONS:
+            'tavg3_2d_lnd_Nx'
+            'inst1_1d_lfs_Nt'
+           ::
+```
+
+To be available for output through MAPL HISTORY, a variable must be defined as an `ExportSpec` in a `GEOS_*GridComp.F90` file. 
+The list of variables ("fields") in each Collection consists of three columns: 
+- (column 1) variable name in `GEOS_[GCNAME]GridComp.F90` file,
+- (column 2) GridComp name [GCNAME], and 
+- (column 3) user-specified variable name that appears in nc4 output (optional).
+
+For example, to write the "WCSF" and "WCRZ" variables (surface and root-zone soil moisture) from the Catchment model ("CATCH") GridComp under the names "SFMC" and "RZMC", respectively, the definition of the `tavg3_2d_lnd_Nx` Collection in `HISTORY.rc` must include the following:
+```
+ tavg3_2d_lnd_Nx.mode:        'time-averaged',
+ tavg3_2d_lnd_Nx.frequency:   030000,
+ tavg3_2d_lnd_Nx.fields:      'WCSF'       , 'CATCH'  , 'SFMC'         ,
+                              'WCRZ'       , 'CATCH'  , 'RZMC'         ,
+                              ::
+```
+
+The same variable can be written in more than one Collection (i.e., at different temporal and/or spatial resolutions), and there is no limit to the number of Collections that are defined for and written by a simulation.  
+
+Gridded ("2d") output can be on a grid other than the "native" grid that is associated with the tile space used in the simulation, as long as the output grid is defined in `HISTORY.rc`.   [_**MORE INFO NEEDED!!!**_]
```

Comment:
@weiyuan-jiang : I never quite understood how exactly the output grid is defined in HISTORY.rc and how it is specified for a given Collection.  As it stands, this section pretty much ignores this aspect, but we'll need to add more info eventually.

Reply:
I have added more info to it.